gravitee-apim-portal-webui-next/src/components/search-bar/search-bar.component.scss

```
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use 'sass:map';
+@import '../../scss/theme/theme';
+
+.search-input {
+  display: flex;
+
+  &__form {
+    margin-left: auto;
+
+    --mdc-filled-text-field-container-color: #{$banner-background-color};
+  }
+}
+
+.mat-form-field-underline {
+  display: none;
+}
```

This would allow all scss related to the search-bar to stay grouped together. It also uses tokens to change the color of the bottom border. Additionally, the `$banner-background-color` doesn't really match figma. I found that the default color is a bit closer to what Figma looks like.

```suggestion
::ng-deep {
  .search-input {
    .mdc-text-field {
      border-radius: 28px;
    }
  }
}

.search-input {
  display: flex;

  &__form {
    margin-left: auto;

    --mdc-filled-text-field-active-indicator-color: transparent;
    --mdc-filled-text-field-focus-active-indicator-color: transparent;
    --mdc-filled-text-field-hover-active-indicator-color: transparent;
    --mdc-filled-text-field-container-shape: 28px;
  }
}
```

---

Done